src/components/members/filters/Filters.js

```diff
@@ -0,0 +1,45 @@
+import {RangeModal} from "./RangeModal";
+import {CHECKBOX_FILTER_MODALS, RANGE_FILTER_MODALS} from "constants/filters";
+import {CheckboxModal} from "./CheckboxModal";
+import {FiltersDropdown} from "./FiltersDropdown";
+
+export const Filters = ({filters, modals}) => {
+
+  const openFilterModal = (key, toggleOpen) => {
+    modals.open(key);
+    toggleOpen(); // close context menu
```

Comment:
Nit: you could just name this second arg `toggleContextMenu` then you have yourself some self documenting code 🖖